vendor/vendor.json

```diff
@@ -1623,6 +1655,22 @@
 			"revision": "2187045faa54fce7f5028706ffeb2f2fc342aa7e",
 			"revisionTime": "2014-05-22T04:34:39Z"
 		},
+		{
+			"checksumSHA1": "SjPjmaPGaIWKtmvyv6cJ7Fdzy50=",
+			"path": "github.com/sanathkr/go-yaml",
+			"revision": "ed9d249f429b3f5a69f80a7abef6bfce81fef894",
+			"revisionTime": "2017-08-19T19:51:28Z",
+			"version": "v2",
+			"versionExact": "v2"
+		},
+		{
+			"checksumSHA1": "qoSnctxGqjdm6K9unlf/Mgo+mbA=",
+			"path": "github.com/sanathkr/yaml",
+			"revision": "0ca9ea5df5451ffdf184b4428c902747c2c11cd7",
+			"revisionTime": "2017-03-27T23:54:44Z",
+			"version": "v1.0.0",
+			"versionExact": "v1.0.0"
+		},
```

Comment:
How come this PR introduces 2 yaml libs?

Reply:
These are two dependencies of goformation they overrides some behavior on the tags.

https://github.com/awslabs/goformation/blob/master/go.mod#L13-L14

Reply:
@urso I wish we didn't have to bring the yaml dependencies, I only use the JSON format :(